3_javascript/javascript1/week2/javascript.js

```diff
@@ -0,0 +1,146 @@
+//Welcome to my second experience in JAVASCRIPT :)
+
+/* console.log("*******     Flight booking fullname function    *******");
+ function getFullname(firstname, surname) {
+     return `${firstname} ${surname}`;
+}
+
+ const fullname1 = getFullname("Helena", "Hashemi");
+ const fullname2 = getFullname("Shahram", "Shokri");
+
+ console.log(fullname1);
+ console.log(fullname2); */
+
+console.log("*******     Flight booking fullname function / ('Formal fullnam')     *******");
+function getFullname(firstname, surname, gender, userFormalName) {
+
+    if(gender === "male") {
+    return `Lord ${firstname} ${surname}`;
+}
+if(gender === "female"){
+    return `Lady ${firstname} ${surname}`;
+}
+if (gender === ""){
+    return `${firstname} ${surname}`;
+}
+}
+
+const userFormalName = getFullname("Soeren", "Sepehr", "");
```

Comment:
if you correct the code above, then you nedd to pass true or false, depending on what value you want to assign to `userFormalName`